From f6e8ed825cfd728b97923061a6d157ee6201fd21 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Tue, 13 Jun 2017 16:29:37 +0300
Subject: [PATCH 175/767] dpaa_eth: ceetm: update include paths for tc filters

The tcf_destroy_chain call prototype has been moved.

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.h b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.h
index e653353270a6..63cc34751d9c 100644
--- a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.h
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.h
@@ -33,6 +33,7 @@
 #define __DPAA_ETH_CEETM_H
 
 #include <net/pkt_sched.h>
+#include <net/pkt_cls.h>
 #include <net/netlink.h>
 #include <lnxwrp_fm.h>
 
-- 
2.17.0

